### PR TITLE
Aws policy branch

### PR DESCRIPTION
**Pull request by JasonBOldham**
I have updated images, added a kb, changed verbiage in some kbs

### CODE REVIEW

**Comment by srveit** [comment]
Is this a duplicate of https://github.com/CenturyLinkCloud/PublicKB/pull/2158 ?
If so close it.

**Comment by JasonBOldham** [comment]
Is there anything I need to do to get this out?

File: Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+**Forecast:**
+To access the anomaly detection and forecasting function you first need to go to the Graphs tab.
+
+
+ ![Graphs Left Nav](../../images/GraphsLeftNav.PNG)
+
+From there you will select your Source and then the metrics you want to view.
+
+![Graph Source](../../images/Graphsource.PNG)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md]: change to `(../../images/cloud-application-manager/GraphSource.PNG)`
[notice the capital "S"]

File: Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+You will see that there is an f(X) button as well as a clipboard icon.  The clipboard icon allows you to copy the metric id to the clipboard.  The _f(X)_ allows you to add the anomaly detection, forecasting, and confidence band functionality.
+ 
+![Graph Functionality](../../images/graphfunctionality.PNG)
+
+To view your original data (CPUUtilization) as well as the functional data you want to apply; you need to add the same dataset to the graph.
+
+![Graphs Adding Data](../../images/graphsaddingdata.PNG)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md]: change to `(../../images/cloud-application-manager/GraphsAddingData.PNG)`
[notice the capitalization]

File: Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Once you have added the second metric you can add your functionality.  The legend will change to show your applied functionality in a new color. 
+
+![Graphs legend change](../../images/graphlegendchange.PNG)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md]: change to `(../../images/Graphlegendchange.PNG)`
[notice the capitalization]

File: Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+_As a rule of thumb, requesting 2 weeks of data generates more meaningful results.  The more data that is available the more accurate you can expect the forecast to be._
+
+![Forecast Graph](../../images/forecastgraph.PNG)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md]: change to `(../../images/Forecastgraph.PNG)`
[notice the capitalization]

File: Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+**Anomaly Detection:**
+
+To access anomaly detection, you would follow the same path as Forecast but would select Anomaly from the f(X) function instead.  
+
+_As a rule of thumb, you need to request at least 72 hours of data to generate more meaningful results.  The more data that is available (up to 2 weeks) the more accurate you can expect the forecast to be._
+
+![Anomaly Graph](../../images/anomalygraph.PNG)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/Anomaly Detection & Forecasting.md]: change to `(../../images/AnomalyGraph.PNG)`
[notice the capitalization]

**Comment by srveit** [comment]
Once you address the above changes, the automated test should pass.

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 6]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
 
 ![MonitoringLeftNavScreenShot](../../images/MonitoringLeftNavScreenShot121317.JPG)
 
-##### Events
+##### [Events](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/events/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `(Events.md)`

See the comment I made in https://github.com/CenturyLinkCloud/PublicKB/pull/2158

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 7]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
...
 On the Events tab there is a list of all active events (i.e. check that are out of their configured bound) for that workspace.  This includes statuses warning and critical, the check-type, its output, count and last occurrence.
 
-##### Policies
+##### [Policies](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/policies/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `(Policies.md)`

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 8]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
...
 Monitoring policies are groupings of one or more server checks available to be applied to a server within the selected workspace. Within the policy itself you can view the server checks and their associated configuration parameters along with the servers currently applied under that policy.  
 
 ##### Agents
 This section lists all of the servers being monitored for the selected workspace. Clicking on an individual agent will provide details specific to that server and also the current status of all checks being reported back from the server.  
 
-##### Suppressions
+##### [Suppressions](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/suppressions/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `(Suppressions.md)`

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 9]
```diff
@@ -31,6 +31,11 @@ For servers that have 'metric' server checks applied to them, for which all Mana
 
 After logging in the user will be in their personal workspace. Click on the workspace switcher to choose the workspace of which agent(s) you wish to view.  
 
+#### [AWS Dashboards](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/awsdashboards/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `(AWSDashboards.md)`

**Comment by srveit** [comment]
Sorry I didn't catch these earlier. :-(

**Comment by JasonBOldham** [comment]
Is this ok to go then?

**Comment by srveit** [comment]
No. You need to address the four comments I made.

**Comment by JasonBOldham** [comment]
I have updated the .md to this pull request

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 10]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
 
 ![MonitoringLeftNavScreenShot](../../images/MonitoringLeftNavScreenShot121317.JPG)
 
-##### Events
+##### [Events](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/Events.md/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `##### [Events](Events.md)`

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 11]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
...
 On the Events tab there is a list of all active events (i.e. check that are out of their configured bound) for that workspace.  This includes statuses warning and critical, the check-type, its output, count and last occurrence.
 
-##### Policies
+##### [Policies](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/Policies.md/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `##### [Events](Policies.md)`

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 12]
```diff
@@ -11,16 +11,16 @@ This document describes how to access the Monitoring UI, and provides a breakdow
...
 Monitoring policies are groupings of one or more server checks available to be applied to a server within the selected workspace. Within the policy itself you can view the server checks and their associated configuration parameters along with the servers currently applied under that policy.  
 
 ##### Agents
 This section lists all of the servers being monitored for the selected workspace. Clicking on an individual agent will provide details specific to that server and also the current status of all checks being reported back from the server.  
 
-##### Suppressions
+##### [Suppressions](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/Suppressions.md/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `##### [Events](Suppressions.md)`

File: Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md [hunk 13]
```diff
@@ -31,6 +31,11 @@ For servers that have 'metric' server checks applied to them, for which all Mana
 
 After logging in the user will be in their personal workspace. Click on the workspace switcher to choose the workspace of which agent(s) you wish to view.  
 
+#### [AWS Dashboards](https://www.ctl.io/knowledge-base/cloud-application-manager/monitoring/AWSDashboards.md/)
```
Comment (srveit) [inline Cloud Application Manager/Monitoring/CTLCloudMonitoringUI.md]: Change this to `##### [Events](AWSDashboards.md)`

**Pull request merged**